examples/python/expanded_two_peers.py

```
@@ -11,10 +11,22 @@
+    except ImportError:
+        from rixl._api import nixl_agent, nixl_agent_config
+        from rixl.logging import get_logger
+
+    logger = get_logger(__name__)
+    NIXL_AVAILABLE = True
+except ImportError:
+    import logging
 
-logger = get_logger(__name__)
+    logger = logging.getLogger(__name__)
+    logger.error("NIXL API missing install NIXL.")
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)